src/deepsparse/open_pif_paf/utils/annotate.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    import openpifpaf
+
+    openpifpaf_error = None
+except ModuleNotFoundError as openpifpaf_import_error:
+    openpifpaf = None
+    openpifpaf_error = openpifpaf_import_error
+
+import logging
+from typing import List, Optional, Tuple
+
+import numpy
+
+import cv2
+import torch
+from deepsparse.open_pif_paf.schemas import OpenPifPafOutput
+from deepsparse.yolact.utils.annotate import (
+    _get_text_size,
+    _plot_fps,
+    _put_annotation_text,
+)
+from openpifpaf import decoder, network
+
+
+_LOGGER = logging.getLogger(__name__)
+
+# necessary openpifpaf dependencies for now
+model_cpu, _ = network.Factory().factory(head_metas=None)
+processor = decoder.factory(model_cpu.head_metas)
```

Comment:
initialization of these must be moved to behind a function otherwise this will run as soon as anything from open pif paf directory is imported

Reply:
I do not understand. What is going to happen is:
1.  `src/deepsparse/open_pif_paf/annotate.py` is ran
2.  during the importing step, `src/deepsparse/open_pif_paf/utils/annotate.py` is being imported and lines 42/43 are evoked. `processor` gets created.
3. until termination, only the `annotate_image` function is called. Note: lines 42/43 never get run again.

Reply:
Yeah I think ben's initial concern won't happen since the `__init__.py` files don't import from sub modules. Another option could be have an Annotator class that creates/holds the processor in constructor, and it just has the `annotate_image` as a method. Might be overkill though. A really hacky option is to store the processor on the function itself (yes you can set attributes on functions lmao):

```python
def annotate_image(...):
    if annotate_image.processor is None:
        annotate_image.processor = ...
    processor = annotate_image.processor
```

Reply:
After consideration, moved the openpifpaf dependencies to my pipeline itself.